### ISSUE TITLE
Interception efficiency tool

### ISSUE THREAD

**Issue by brunopitton** [skp703/CW3#6]
One can enter distances between centers/rows that are smaller than the diameter of the pot which results in interception efficiency greater than 100%. It would be good for growers to be able to enter "spacing". Typically a grower is spacing plants equally on all sides to ensure uniform growth, i.e. 2-foot centers. So all plants are 2 feet away in any direction. I see this tool being used for planning, not as the plants are pulled for sale and a row is missing, which is the only scenario I think one would use for entering two spacing dimensions.

**Comment by skp703** [comment]
Yeah, I noticed that bug too. Maybe we need spacing. We should discuss with John.
